resources/definitions/fdmprinter.def.json

```diff
@@ -1228,7 +1228,8 @@
                     {
                         "back": "User Specified",
                         "shortest": "Shortest",
-                        "random": "Random"
+                        "random": "Random",
```

Comment:
I think it would be a good idea to also rename `Shortest` to `Shortest Path` even though it's a bit unrelated to the feature you are introducing...